### PR TITLE
Azure resource manager (ARM) support

### PR DESCRIPTION
**Pull request by stoader**

### CODE REVIEW

**Review by matyix — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,9 +1,13 @@
...
 The control plane image (AMI) is currently published to one region, `eu-west-1` aka Ireland. When launching the control plane please pass the following *ImageId* `ami-c070c0b9`.
 
-The template is accessible from the [following](https://s3-eu-west-1.amazonaws.com/cf-templates-grr4ysncvcdl-eu-west-1/2017338aEC-new.template6z1wxm2h6cb) location.
+The template is accessible from the [following](https://s3-eu-west-1.amazonaws.com/cf-templates-grr4ysncvcdl-eu-west-1/2018026em9-new.templatee93ate9mob7) location.
 
 In case of using the `Makefile` use the `.env.example` as a start.
+
+# Pipeline control plane launcher on Azure
+
+Please see [how to luanch pipeline control plane on Azure](https://github.com/banzaicloud/pipeline/blob/0.2.0/docs/pipeline-howto.md#launch-pipeline-control-plane-on-azure) for details.
```
Comment (baluchicken) [inline README.md]: Typo error in launch

File: control-plane-cf.template [hunk 2]
```diff
@@ -420,15 +409,15 @@ Resources:
...
-            - |2
+            - |
                 helm repo add banzaicloud-stable http://kubernetes-charts.banzaicloud.com
-            - |2
+            - |
                 helm repo update
-            - |2
+            - |
                 helm repo list
             - |2
```
Comment (baluchicken) [inline control-plane-cf.template:422]: Why did you remove the other |2 above and leave this one intact?

**Review by baluchicken — CHANGES_REQUESTED** [review]
Some minor things.

File: control-plane-cf.template (hunk 2 above)
Comment (kozmagabor) [reply control-plane-cf.template:422]: We don't need two "space" indent in these lines. So it's ok, if we remove it.

Comment (baluchicken) [reply control-plane-cf.template:422]: Thanks for the clarification:)

**Review by baluchicken — APPROVED** [review]

**Review by stoader — COMMENTED** [review]
LGTM

**Review by matyix — APPROVED** [review]
thx @kozmagabor @stoader lgtm
